src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -110,7 +110,7 @@ const NavSearchBar = () => {
           key={`user-${user.id}`}
           title={user.partyName ?? DEFAULT_PARTY_NAME}
           image={user.pictureUrl}
-          isAvatar={true}
+          user={user}
```

Comment:
Since we're passing the entire `user` object in here now, and `image` is an optional field, presumably we can remove that prop here entirely?

Reply:
fixed: https://github.com/sparkletown/sparkle/pull/1350/commits/ce81df56da1ee5e60ad3e28b8d7ad395f7556447